server/routerlicious/packages/lambdas-driver/src/document-router/lambdaFactory.ts

```diff
@@ -16,7 +16,7 @@ import { DocumentLambda } from "./documentLambda";
 /**
  * @internal
  */
-export class DocumentLambdaFactory extends EventEmitter implements IPartitionLambdaFactory {
+export class DocumentLambdaFactory<T> extends EventEmitter implements IPartitionLambdaFactory<T> {
```

Comment:
Nit: if the type parameter is specifically for the config, might be better to call it `TConfig`. Would also recommend adding any relevant type documentation via an `@typeParam` comment above.